boards/arc/nsim/board.cmake

```diff
@@ -8,21 +8,32 @@ endif()
 
 if(${CONFIG_SOC_NSIM_EM})
 board_runner_args(arc-nsim "--props=nsim_em.props")
+set(NSIM_PROPS nsim_em.props)
 board_runner_args(mdb-nsim "--nsim_args=mdb_em.args")
+set(MDB_ARGS mdb_hs_smp.args)
```

Comment:
With the NSIM_PROPS and MDB_ARGS variables, I suggest to factor out the 
    board_runner_args(arc-nsim "--props=nsim_em.props")
    board_runner_args(mdb-nsim "--nsim_args=mdb_em.args")

and replace them with a single copy using the variables. Something like
    board_runner_args(arc-nsim "--props=${NSIM_PROPS }")
    board_runner_args(mdb-nsim "--nsim_args="${MBD_ARGS}")

Reply:
I like the change for this, makes it even more compact and scalable, instead of all the hardcoded board names.